### PR TITLE
require python 3.5 as minimum version

### PR DESCRIPTION
**Pull request by poettering**
Fixes: #79

### CODE REVIEW

File: mkosi [hunk 1]
```diff
@@ -22,6 +22,9 @@ from enum import Enum
 
 __version__ = '1'
 
+if sys.version_info[1] < 5:
```
Comment (keszybz) [inline mkosi]: That's unpythonic, e.g. python 2.7 passes that test. (And a hypothetical python 4.0 would also fail the test...). I'd prefer a check like:
```python
if sys.version_info < (3, 5):
```

Comment (poettering) [inline mkosi]: well, we specify "python3" in the shebang header, hence the major is already checked, i only checked the minor...

I am not sure I fully grok tuple comparisons in python3, for example, is (3, 5, 0) < (3, 5) or is (3, 5) < (3, 5, 0)? I am not keen on using expressions whose full effect I really don't grok...

Comment (keszybz) [inline mkosi]: They are sorted in "dictionary" order, i.e. pairs of elements in the same position are compared, and if they are all equal, the longer sequence is later. This comparison is a standard Python idiom, in particular used quite often for versions ;)

**Comment by poettering** [comment]
posted a new version now. Please have a look

**Pull request merged**